scio-tensorflow/src/test/scala/com/spotify/scio/io/TFTapTest.scala

```diff
@@ -19,10 +19,26 @@ package com.spotify.scio.io
 
 import java.util.UUID
 
+import com.spotify.scio.ScioContext
+import org.apache.beam.sdk.Pipeline.PipelineExecutionException
+import org.apache.beam.sdk.io.TFRecordIO.CompressionType
 import org.apache.commons.io.FileUtils
+import org.tensorflow.example._
 
 class TFTapTest extends TapSpec {
 
+  private def getDummyExample = {
+    val features = Seq((1.0F, 2.0F), (5.0F, 3.0F))
+    features.map { kv =>
```

Comment:
Use a `TensorFlowType[T]` here as well to simplify the code? We should probably expect the users to do that too right?